modules/apigee_edge_teams/src/Plugin/EntityReferenceSelection/TeamMembersSelection.php

```diff
@@ -222,4 +229,21 @@ protected function buildEntityQuery($match = NULL, $match_operator = 'CONTAINS')
     return $query;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getReferenceableEntities($match = NULL, $match_operator = 'CONTAINS', $limit = 0) {
+    $options = parent::getReferenceableEntities($match, $match_operator, $limit);
+
+    $target_type = $this->getConfiguration()['target_type'];
+    $ids = array_keys($options[$target_type]);
+    $entities = $this->entityTypeManager->getStorage($target_type)->loadMultiple($ids);
+
+    /* @var \Drupal\user\UserInterface $entity */
+    foreach ($entities as $id => $entity) {
+      $options[$target_type][$id] = Html::escape($entity->getEmail());
```

Comment:
Why do you escape email addresses? Compared with a parent where a label() could be any kind of user input, I think we can safely assume that stored email addresses are always valid which mains they can not contain an XSS string.

Reply:
Good catch.